react/tic-tac-toe/src/app/screens/Game/index.js

```diff
@@ -1,18 +1,36 @@
 import React, { Component } from 'react';
 
+import matchesService from '../../../services';
+
 import styles from './styles.module.scss';
 import Board from './components/Board';
+import Matches from './components/Matches';
 import { calculateWinner } from './utils';
 
+
 class Game extends Component {
   state = {
     history: [{
       squares: Array(9).fill(null)
     }],
     stepNumber: 0,
-    xIsNext: true
+    xIsNext: true,
+    matches: [],
+    isLoading: false
   };
 
+  async componentDidMount() {
+    const response = await matchesService.getMatches();
```

Comment:
we usually dont make service calls from redux's actions, so there should be a matchesActions which calls this service, and then exposes the matches data via a reducer. Sorry i didnt mention this in the previous comment.